### PR TITLE
arabica: fix addition of test do block

### PR DESCRIPTION
**Pull request by TanyaaCJain**
- [y] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [y] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [y] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [n] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/arabica.rb [hunk 1]
```diff
@@ -28,4 +28,9 @@ def install
     system "make", "check" if build.with? "test"
     system "make", "install"
   end
+  
+  test do
+    (testpath/"test.file").write("writing some test file, if you need to\n")
+    system bin/"arabica", "-o"
```
Comment (fxcoudert) [inline Formula/arabica.rb]:
```
==> /usr/local/Cellar/arabica/20160214/bin/arabica -o
Failed to execute: /usr/local/Cellar/arabica/20160214/bin/arabica
Error: arabica: failed
```

File: Formula/arabica.rb [hunk 2]
```diff
@@ -28,4 +28,9 @@ def install
     system "make", "check" if build.with? "test"
     system "make", "install"
   end
+  
+  test do
+    (testpath/"test.file").write("writing some test file, if you need to\n")
```
Comment (fxcoudert) [inline Formula/arabica.rb]: Don't.

File: Formula/arabica.rb [hunk 3]
```diff
@@ -28,4 +28,9 @@ def install
     system "make", "check" if build.with? "test"
     system "make", "install"
   end
+
```
Comment (fxcoudert) [inline Formula/arabica.rb]: `C: 31:  1: Trailing whitespace detected.`

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by dunn** [comment]
Ping on the above comments!

**Comment by fxcoudert** [comment]
No response, closing this PR for now.

**Pull request closed without merging**